elm.json

```diff
@@ -1,33 +1,33 @@
 {
-  "type": "package",
```

Comment:
You can ignore these changes. They're a result of publishing under a different name, and whitespace changes from how the `elm` commands format the file.